tema_3/main.py

```diff
@@ -0,0 +1,70 @@
+import json
+# import random
+import csv
+import os
+
+# generate a folder output_data with multiple .json files
+output_folder = "output_folder"
+if not os.path.exists(output_folder):
+    os.makedirs(output_folder)
+
+
+if __name__ == '__main__':
+
+    # read the data from file input.csv
+    with open("input.csv") as f:
```

Comment:
You should just read the data from the opened file, you don't have to do the whole functionality within the `with` instruction.